### ISSUE TITLE
"New device found in your network: undefined (undefined)

### ISSUE THREAD

**Issue by jerrchen** [firewalla/firewalla#421]
This is a ipv6 only device. so there is no ipv4 address.  

----------------- 

1) "aid"
 2) "35"
 3) "type"
 4) "ALARM_NEW_DEVICE"
 5) "device"
 6) "undefined"
 7) "alarmTimestamp"
 8) "1508393104.48"
 9) "timestamp"
10) "1508393104.479"
11) "notifType"
12) "security"
13) "p.device.vendor"
14) "Intel Corporation"
15) "p.device.mac"
16) "00:50:F1:80:00:00"
17) "p.device.ip"
18) "undefined"
19) "p.device.name"
20) "undefined"
21) "p.device.id"
22) "undefined"
	 
	 INFO 2017-10-19 06:07:03 IPv6DiscoverySensor: Found V6 Address  [ 'fe80::250:f1ff:fe80:0' ] 00:50:F1:80:00:00
		 
		 127.0.0.1:6379> hgetall host:mac:00:50:F1:80:00:00
 1) "ipv6Addr"
 2) "[\"fe80::250:f1ff:fe80:0\"]"
 3) "mac"
 4) "00:50:F1:80:00:00"
 5) "firstFoundTimestamp"
 6) "1508393103.522"
 7) "lastActiveTimestamp"
 8) "1508393103.522"
 9) "macVendor"
10) "Intel Corporation"
11) "bname"
12) "undefined"
13) "bnameCheckTime"
14) "1508393104"

**Comment by MelvinTo** [comment]
deferred, not able to reproduce

**Comment by jerrchen** [comment]
This happens when devices with ipv6 only address is found.

**Comment by MelvinTo** [comment]
right, not able to reproduce with ipv6 only. also from code logic, it's not
likely to trigger this issue with ipv6 only.
